src/components/calcite-combobox/calcite-combobox.scss

```diff
@@ -137,21 +138,18 @@
 }
 
 .list-container {
-  overflow-y: auto;
-  max-height: 100vh;
+  @apply overflow-y-auto
+    max-h-screen;
   width: var(--calcite-dropdown-width);
   background: var(--calcite-ui-foreground-1);
```

Comment:
`bg-foreground-1` I think

Reply:
I accidentally have Combobox in this pr when it's already here: https://github.com/Esri/calcite-components/pull/1789

That said, I did update this in the Combobox pr. thx dude!